lib/commands/bootstrap.js

```diff
@@ -25,19 +24,22 @@ module.exports = {
 		return new Promise( ( resolve, reject ) => {
 			const destinationPath = path.join( data.options.packages, data.repository.directory );
 
-			// Package is already cloned.
-			if ( fs.existsSync( destinationPath ) ) {
-				log.info( `Package "${ data.packageName }" is already cloned. Skipping.` );
+			let promise = Promise.resolve();
 
-				return resolve( { logs: log.all() } );
+			// Package is not cloned.
+			if ( fs.existsSync( destinationPath ) ) {
+				log.info( `Package "${ data.packageName }" is already cloned.` );
+			} else {
+				const command = [
```

Comment:
Since this condition sets the promise to `exec( cmd )`, the previous condition could set it to `Promise.resolve()`. This would be clearer.